### DISCUSSION TITLE
How to deal with raw pointers that is not Send?

### DISCUSSION THREAD

**Discussion by cenwangumass** [slawlor/ractor#195] [Q&A]
I would like to spawn multiple actors where each of them holds a `type State = Model`. However, it is currently complain that a field of Model does not implement Send. In fact this field is a Rust struct contains a raw pointer from a C lib. What should I do in this case?

**Comment by slawlor** [comment]
Just make your `Model` implement Send

```rust
unsafe impl Send for Model {}
```

**Reply by slawlor** [reply]
Assuming it's actually thread safe 😁
